res/openvpn-gui-res-no.rc

```diff
@@ -371,29 +370,29 @@ BEGIN
     IDS_ERR_SOCKS_PROXY_PORT "SOCKS proxy port må oppgis."
     IDS_ERR_SOCKS_PROXY_PORT_RANGE "SOCKS proxy port må være ett tall mellom 1 og 65535"
     IDS_ERR_CREATE_REG_HKCU_KEY "En feil oppsto ved opprettelse av register-nøkkel ""HKEY_CURRENT_USER\\%s"""
-    IDS_ERR_GET_TEMP_PATH "En feil oppsto når GetTempPath() ble kalt opp. Alternativ ""C:\\"" vil bi benyttet."
+    IDS_ERR_GET_TEMP_PATH "En feil oppsto når GetTempPath() ble kalt opp. Alternativ ""C:\\"" vil bli benyttet."
```

Comment:
I think this is no longer used.  Temporary files directory or temporary files folder?